docs/source/intro-clx-loda-anomaly-detection.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>can you clarify that each row is given an anomaly score between -inf and inf with larger scores indicating more anomalous data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The scores is supposed to be b/n 0 &amp;+inf, as we are taking negative log likelihood values as score.</p><p> </p>